website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,288 @@
+
+However, there are use cases where a `cluster-scoped` operator may make sense. For example, the [cert-manager][cert-manager] operator is often deployed with cluster-scoped permissions and watches so that it can manage to issue certificates for an entire cluster.
+
+**IMPORTANT**: When a [Manager][ctrl-manager] instance is created in the `main.go` file, it might receives the namespace(s) as Options as described bellow. These namespace(s) should be watched and cached for the Client which is provided by the Controllers. Only clients provided by `cluster-scoped` projects are able to manage `cluster-scoped` CRD's. For further information see: [CRD scope doc][crd-scope-doc].
+
+## Watching and Managing resources in all cluster (Default)
+
+An [Manager][ctrl-manager] is responsible for manage the controllers of the operator project. In the `main.go` you will be able to find it. See:
+
+```go
+    ...
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme:             scheme,
+		MetricsBindAddress: metricsAddr,
+		Port:               9443,
+		LeaderElection:     enableLeaderElection,
+		LeaderElectionID:   "f1c5ece8.example.com",
+	})
+	if err != nil {
+		setupLog.Error(err, "unable to start manager")
+		os.Exit(1)
+	}
+    ...
+```
+
+**NOTE** The above example is the default code generated for the projects created with `operator-sdk init` which are creating a new Manager for an operator `cluster-scoped`.
+
+## Watching and Managing resources in an Namespace
+
+To restrict the scope of the [Manager's][ctrl-manager] cache to an specific `namespace` is required to inform it via the `Namespace` attribute from [Options][ctrl-options]. See: 
+
+```go
+    ...
+    mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+        Scheme:             scheme,
+        Namespace:          "operator-namespace",
+        MetricsBindAddress: metricsAddr,
+    })
+    ...
+``` 
+
+## Watching and Managing resources in a set of Namespaces
+
+It is possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] attribute from [Options][ctrl-options] to watch and manage resources only in a specific set of namespaces. See:
+
+```go
+    ...
+    var namespaces []string // List of Namespaces
+
+    mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+        Scheme:             scheme,
+        NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+        MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+    })
+    ...
+```
+
+In the above example, if a CR(CustomResource) which is managed by its controllers be applied in a namespace that is not in this List of `namespaces` then, its controller will not re-trigger its reconcile since the namespaced is NOT part of the [Manager][ctrl-manager] scope.
```

Comment:
rephrase: if a CR(CustomResource) which is managed by its controllers be applied in a namespace. ->
if a CR(CustomResource), which is managed by its controllers, is applied to a namespace